src/pages/index.js

```diff
@@ -5,45 +5,83 @@ import {
   disableButton,
   resetValidation,
 } from "../scripts/validation.js";
```

Comment:
* There is an error in the console https://snipboard.io/Uy5gzM.jpg

In `validation.js`, you should add a check inside `setEventListeners `


```js

const setEventListeners = (formEl, config) => {
  const inputList = Array.from(formEl.querySelectorAll(config.inputSelector));
  const buttonEl = formEl.querySelector(config.submitButtonSelector);
  if (!buttonEl) return;  // if there is no button, then stop the function
  toggleButtonState(inputList, buttonEl, config);

  inputList.forEach((inputEl) => {
    inputEl.addEventListener("input", function () {
      checkInputValidity(formEl, inputEl, config);
      toggleButtonState(inputList, buttonEl, config);
    });
  });
};
```